go/tasks/plugins/hive/executor.go

```diff
@@ -150,7 +150,7 @@ func NewQuboleHiveExecutor(ctx context.Context, cfg *config.Config, quboleClient
 	return QuboleHiveExecutor{
 		id:              quboleHiveExecutorID,
 		cfg:             cfg,
-		metrics:         getQuboleHiveExecutorMetrics(scope),
+		metrics:         getQuboleHiveExecutorMetrics(scope.NewSubScope("hive")),
```

Comment:
Reminder to make sure non of the existing alerts are based on the renamed metric or to fix them if they are

Reply:
The metrics below were never added. It is added in this PR.

Reply:
This will affect the existing ones tooThe resourceWaitTime